lib/convection/control/stack.rb

```diff
@@ -248,6 +275,22 @@ def validate
         puts "\nTemplate validated successfully"
       end
 
+      def after_create_task(task)
+        @tasks[:after_create].unshift(task)
```

Comment:
if i define 2 tasks:

```
stack test, MY::TEST::STACK do
  after_create_task(a)
  after_create_task(b)
end
```

then the order of task execution would be `b` then `a`?